README.md

```diff
@@ -52,4 +64,32 @@ After code runs, you can view the data in a Jupyter notebook at `getting_started
 
 <h2>Running the Tests</h2>
 
-To run the tests, run `pytest` from the root directory. Also see the [GitHub Actions](.github/workflows/run_tests.yaml) for the full list of tests that run in continuous integration.
+1. To run the unit tests, run `pytest` from the root directory. 
+2. Run `flake8` from the root directory to check that code style rules have been followed.
+3. Run `mypy src --ignore-missing-imports` and `mypy tests --ignore-missing-imports` from the root directory to check for type errors.
+4. Also see the [GitHub Actions](.github/workflows/run_tests.yaml) for the full list of tests that run in continuous integration.
+
+<h2>Commiting changes</h2>
```

Comment:
I think instructions for Git are out of the scope of this document. Is there any existing documentation like "Git for Beginners" we could link to here instead?

Reply:
I've linked to https://guides.github.com/ in the latest revision.